.prettierrc

```diff
@@ -1,3 +1,7 @@
 {
-  "singleQuote": false
+  "singleQuote": false,
```

Comment:
Revert this change.  
If we want to update `.prettierrc`, let's do it in a discrete PR where we apply the changes to the whole project.  
Adding it with another, will cause future PR's to have a lot of unrelated formatting changes.

Reply:
leaving this as discussed to enforce 'defaults'.